.teamcity/settings.kts

```diff
@@ -49,7 +49,9 @@ val targets = arrayOf(
         "windows/amd64/tip",
 
         "mac/amd64/1.16",
-        "mac/amd64/tip"
+        "mac/amd64/tip",
+
+        "mac/arm64/1.16"
```

Comment:
Can you please also add tip for the arm builder.

Reply:
done